### PR TITLE
Change to msporny di proof

### PR DESCRIPTION
**Pull request by aljones15**
Refactors tests to use the `Data Integrity` assertion, implementations library, etc.

### CODE REVIEW

File: credentials-generator/helpers.js [hunk 1]
```diff
@@ -33,20 +32,14 @@ const cloneJSON = data => JSON.parse(JSON.stringify(data, null, 2));
  *
  * @returns {Promise<object>} - Returns the resulting did key driver result.
  */
-const getDiDKey = async ({seedMultiBase = _seed} = {}) => {
+const getDiDKey = async ({
```
Comment (msporny) [inline credentials-generator/helpers.js:35]:
```suggestion
const getDidKey = async ({
```

`DiD` is incorrect (it doesn't follow a consistent naming pattern)... It's either `getDidKey` or `getDIDKey`... and I think we prefer the former.

File: credentials-generator/index.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ */
+
+const vc = require('@digitalbazaar/vc');
+const canonicalize = require('canonicalize');
+const {createSign, generateKeyPair} = require('crypto');
+const base58btc = require('base58-universal');
+const {join} = require('path');
+const {promisify} = require('util');
+const {
+  cloneJSON,
```
Comment (msporny) [inline credentials-generator/index.js:12]:
```suggestion
  cloneJson,
```

File: credentials-generator/helpers.js [hunk 3]
```diff
@@ -55,10 +45,8 @@ const ISOTimeStamp = ({date = new Date()} = {}) => {
 };
 
 module.exports = {
-  getDiDKey,
-  cloneJSON,
-  writeJSON,
-  getInvocationSigner,
+  getDidKey,
+  writeJson,
   ISOTimeStamp
```
Comment (msporny) [inline credentials-generator/helpers.js:50]:
```suggestion
  IsoTimestamp
```

File: credentials-generator/index.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  return {path: `${credentialsPath}/digestSha512.json`, data: signedVC};
+}
+
+function _validVC() {
+  return {path: `${credentialsPath}/validVC.json`, data: klona(credential)};
+}
+
+async function _issuedVC(key) {
+  const suite = new Ed25519Signature2020({key});
+  const signedVC = await vc.issue({
```
Comment (msporny) [inline credentials-generator/index.js:117]:
```suggestion
  const signedVc = await vc.issue({
```

File: credentials-generator/index.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    credential: klona(credential),
+    suite,
+    documentLoader
+  });
+  return {path: `${credentialsPath}/issuedVC.json`, data: signedVC};
```
Comment (msporny) [inline credentials-generator/index.js:122]:
```suggestion
  return {path: `${credentialsPath}/issuedVC.json`, data: signedVc};
```

Please change everywhere.

File: tests/10-create.js [hunk 6]
```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const chai = require('chai');
+const {implementations} = require('vc-api-test-suite-implementations');
+const {validVC} = require('../credentials');
```
Comment (msporny) [inline tests/10-create.js:8]:
```suggestion
const {validVc} = require('../credentials');
```

File: tests/10-create.js [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+const should = chai.should();
+const bs58 = /^[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]+$/;
+
+// multiple test suite names violate max-len
+/* eslint-disable max-len */
+
+describe('Ed25519Signature2020 (create)', function() {
+  for(const [name, implementation] of filtered) {
+    let verifier;
+    let issuedVC;
```
Comment (msporny) [inline tests/10-create.js:31]:
```suggestion
    let issuedVc;
```

File: tests/10-create.js [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+    let proofs;
+    before(async function() {
+      const issuer = implementation.issuers.find(issuer =>
+        issuer.tags.has('Ed25519Signature2020'));
+      verifier = implementation.verifiers.find(verifier =>
+        verifier.tags.has('VC-HTTP-API'));
```
Comment (msporny) [inline tests/10-create.js:37]:
```suggestion
        verifier.tags.has('VC-API'));
```

This should probably be all lowercase... and might need to be ['vc-verifier-api'] or ['vc-api', 'verifier'] ... probably the former. I understand that there is a problem with stuff like 'Ed25519Signature2020' ... in the future, the cryptosuites are going to be all lowercase e.g., 'eddsa-2022'

Comment (aljones15) [reply tests/10-create.js:37]: this one will require changes in other libraries and will cause issues with other tests suites as this is set in implementations: https://github.com/w3c-ccg/vc-api-test-suite-implementations/tree/initial/implementations

We might want to make an issue on implementations and then update all test suites that use that tag (I believe the verifier tests also use it).

p.s. we need consistent and agreeable tags for implementations.

**Review by msporny — APPROVED** [review]

**Pull request merged**